pydoctor/epydoc/markup/__init__.py

```diff
@@ -352,3 +375,79 @@ def __repr__(self) -> str:
             return '<ParseError on unknown line>'
         else:
             return f'<ParseError on line {self._linenum + 1:d}>'
+
+class SummaryExtractor(nodes.NodeVisitor):
```

Comment:
Maybe mark it as private ... and I guess that maxchars argument is fine, but it can also be a private (global) constant ... I don't see how the maxchars can be used now. 

```suggestion
class _SummaryExtractor(nodes.NodeVisitor):
```

Reply:
I've marked it as public because it's used in another module.